sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -906,4 +906,44 @@ object DateTimeUtils {
     val days = period.getDays
     new CalendarInterval(months, days, 0)
   }
+
+  /**
+   * Since the Proleptic Gregorian calendar is de-facto calendar worldwide, as well as the chosen
+   * one in ANSI SQL standard, Spark 3.0 switches to it by using Java 8 API classes. However, the
```

Comment:
It's a bit confusing to say java 7 & 8 as the old APIs are also available in java 8.

How about `SimpleDateFormat` and `DateTimeFormatter`?

Reply:
Thanks, done in e846fbb.